docs/content/using/changing_node_connectivity.md

```diff
@@ -0,0 +1,18 @@
+# Changing node connectivity
+
+Node connectivity for dynamic host definition needs to be updated when node connectivity changes take place.
+
+Before you begin, ensure that the old host definition is deleted.
+
+Use the following procedure to redefine host connectivity:
+
+1. Undeploy the CSI node pod from the relevant node that the host definition is a part of.
+2. Verify that all host definition instances of the node are deleted.
+     
+          $> kubectl get hostdefinition | grep <node-name> | wc -l
+     
+     The output should be `0`.
+3. From the host, change the host connectivity type.
```

Comment:
@matancarmeli7 [gentle reminder](https://github.com/IBM/ibm-block-csi-driver/pull/517#discussion_r911035386)

Reply:
yes we can remove the word "type", which means that we change the connectivity and it includes changing connectivity ports(in the same type) and changing the connectivity ports (in different type)

Reply:
updated to just "change the host connectivity" but per the new section name - should it be "change the node connectivity" or leave as is?
@oriyarde  @matancarmeli7

Reply:
I think the line and the file name should be aligned, but I think both need to have node and not host, @oriyarde what do you think about changing all the "host" to "node" in this sentence?

Reply:
sure, except when "host" is right before "definition"